stable/database/values.yaml

```diff
@@ -402,6 +402,37 @@ database:
   isManualVolumeProvisioning: false
   isRestore: false
 
+  # NuoDB supports explicit database protocol version upgrade which is performed
+  # after upgrading the NuoDB image for all database processes. To simplify
+  # NuoDB version rollout, Kubernetes Aware Admin (KAA) is used to automatically
+  # upgrade database protocol and restart a Transaction Engine (TE) as an
+  # upgrade finalization step. For more information on how to perform the steps
+  # manually, please check Upgrading the Database Protocol
+  # (https://doc.nuodb.com/nuodb/latest/deployment-models/physical-or-vmware-environments-with-nuodb-admin/installing-nuodb/upgrading-to-a-new-release/upgrading-the-database-protocol/).
+  #
+  # The database protocol version is eligible for an upgrade to the max possible
+  # version if all of the bellows are true:
+  # - the database is in RUNNING state. This ensures that all SMs/archives are
+  #   RUNNING.
+  # - all database processes are MONITORED
+  # - the automatic upgrade is enabled for this database (disabled by default)
+  # - all database processes are running with the same release binary version
+  # - there are available protocol versions for upgrading to
```

Comment:
is this an implementation detail of KAA? Should we stick this text in the NuoDB docs instead? It can change independently of the Helm repo and the commend can become misleading/wrong.